src/ui/server/controllers/insurance/policy-and-export/multiple-contract-policy/validation/rules/maximum-buyer-will-owe.test.ts

```diff
@@ -108,4 +108,16 @@ describe('controllers/insurance/policy-and-export/multiple-contract-policy/valid
       expect(result).toEqual(mockErrors);
     });
   });
+
+  describe('when there are no validation errors and the value contains a comma', () => {
+    it('should return the provided errors object', () => {
+      const mockSubmittedData = {
+        [FIELD_ID]: '10,000',
```

Comment:
Feels a bit overkill for the unit tests because the `isWholeNumber` helper validation checks this and we already have 100% test coverage. Added additional tests though, can't hurt.